Druid-Tests/DruidTestInterpreter.class.st

```diff
@@ -734,58 +734,54 @@ DruidTestInterpreter >> primitiveFloat64AtPut [
 
 { #category : #'as yet unclassified' }
 DruidTestInterpreter >> primitiveFloatAdd [
-
 	"Transform the integers as float.
 	Add the floats.
 	Then read the result raw representation and push that to the stack."
 
 	<numberOfArguments: 1>
-	self
-		pop: 2
-		thenPush:
-		(self stackTop asFloat + (self stackValue: 1) asFloat) rawFloatBits
+	self pop: 2 thenPush: (objectMemory
+			 rawFloatBitsOf:
+			 self stackTop asFloat + (self stackValue: 1) asFloat
+			 in: nil)
 ]
```

Comment:
You see, this reads much better (and the version on the right has that strange `nil` argument that seems unused)a...

Reply:
https://github.com/Alamvic/druid/issues/23